### PR TITLE
syncer: improve default skip SQL rules

### LINKED ISSUE #121: Syncer: parse sql error like `drop procedure`
If such sqls are unsupported, they should be skipped. Don't make program crash.

```
delimiter $$
drop procedure if exists prepare_data_1_1;
create procedure prepare_data_1_1()
begin
    declare i int default 8000001;
    while i < 8100001 do
        insert into wallet_user_trans_0(id,a, b) values(
            i,
            case when i % 300000 = 0 then null else i end,
            case when i % 300000 = 0 then null else md5(concat('', i)) end
        );
        set i = i + 1;
    end while;
end $$
delimiter ;
set autocommit=false;
start transaction;  
call prepare_data_1_1();
commit;
```

### PR DESCRIPTION
**Pull request by holys**
this fix #121 ,  and enhance it

refer: https://dev.mysql.com/doc/refman/5.7/en/sql-syntax.html

@GregoryIan  PTAL

### CODE REVIEW

File: syncer/filter.go [hunk 1]
```diff
@@ -14,18 +14,41 @@ var (
...
-		"^ALTER\\s+USER",
+		"^ALTER\\s+VIEW",
+
+		//function
+		"^CREATE\\s+FUNCTION",
+		"^ALTER\\s+FUNCTION",
+		"^DROP\\s+FUNCTION",
+
+		// account management
+		"^GRANT",
```
Comment (GregoryIan) [inline syncer/filter.go]: Doesn't tidb support GRANT?

**Comment by GregoryIan** [comment]
LGTM

File: syncer/filter.go (hunk 1 above)
Comment (andelf) [inline syncer/filter.go]: limited support for GRANT, better turn it off

**Comment by holys** [comment]
@andelf  @nolouch  PTAL

**Comment by nolouch** [comment]
LGTM

**Pull request merged**